integrations/intercom/src/index.ts

```diff
@@ -192,6 +192,7 @@ const integration = new Integration({
           id: conversationId,
           admin_assignee_id: adminAssigneeId,
           conversation_parts: { conversation_parts },
+          source: firstConversationPart,
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏They refer it as conversation source in the docs